pkg/assetfetcher/http.go

```diff
@@ -52,6 +51,10 @@ type NexposeAssetFetcher struct {
 	HTTPClient *http.Client
 	// The Nexpose host that points to your instance
 	Host string
+	//
```

Comment:
Nit: looks like this is missing a comment

Reply:
Yep, sure is. Thanks for pointing that out. will fix.